conftest.py

```
@@ -639,7 +639,12 @@ def precopy_interval_forkliftcontroller(
         ensure_exists=True,
     )
 
-    snapshots_interval = str(py_config["snapshots_interval"])
+    try:
+        snapshots_interval = int(py_config["snapshots_interval"])
+    except (TypeError, ValueError) as exp:
+        raise ValueError(
+            f"Config 'snapshots_interval' must be numeric, got {py_config.get('snapshots_interval')!r}"
+        ) from exp
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)